diff --git a/node_modules/laravel-mix/src/webpackPlugins/BuildOutputPlugin.js b/node_modules/laravel-mix/src/webpackPlugins/BuildOutputPlugin.js
index 241a2ed..07b9301 100644
--- a/node_modules/laravel-mix/src/webpackPlugins/BuildOutputPlugin.js
+++ b/node_modules/laravel-mix/src/webpackPlugins/BuildOutputPlugin.js
@@ -3,7 +3,7 @@ const chalk = require('chalk');
 const Table = require('cli-table3');
 const readline = require('readline');
 const stripAnsi = require('strip-ansi');
-const { formatSize } = require('webpack/lib/SizeFormatHelpers');
+const formatSize = require('webpack/lib/util/formatSize');
 const { version } = require('../../package.json');
 
 /**
